src/mjolnir/speed_assigner.h

```diff
@@ -65,6 +65,7 @@ The json basically looks like this:
 
 constexpr uint32_t kMinSuburbanDensity = 5;
 constexpr uint32_t kMaxSuburbanDensity = 10;
+constexpr uint32_t kUnconfiguredSpeed = 0;
```

Comment:
sentinal value that indicates no configured speed exists for a particular directed edge